addons/web/static/src/views/pivot/pivot_model.js

```diff
@@ -1089,12 +1089,9 @@ export class PivotModel extends Model {
             if (field.type === "many2one") {
                 field.group_operator = "count_distinct";
             }
-            if (field.group_operator === undefined) {
-                throw new Error(
-                    "No aggregate function has been provided for the measure '" + measure + "'"
-                );
+            if (field.group_operator !== undefined) {
```

Comment:
I think that the fix should be done in computeReportMeasures instead: in
```
        if (["integer", "float", "monetary"].includes(field.type)) {
            measures[fieldName] = field;
        }
```
one should add the condition field.group_operator.
The fix should contain a simple test that check the content of the menu Mearsures with a field of type currency  with no group_operator.

Reply:
adapt the commit message accordingly